### DISCUSSION TITLE
API to find/filter courses having certificates

### DISCUSSION THREAD

**Discussion by paritshivani** [Sunbird-Lern/Community#187] [Query]
Hello team
Hope you are doing well

Background -
We have updated few screens for sunbird-Ed and trying to add few more details

Issue Details -
Is there any API available which can tell me weather course or its batch have certificate?. I wanted to find total count of certificates achieved in total number of courses having certificates from my enrolled list

@komalm, @manishakapadnis, @AmiableAnil

**Comment by ManojNathIC** [comment]
Hello team, 

Just following up on the above
 
Any guidance on this would be greatly appreciated.

Thank you!
